src/less/icon/generated/icon.less

```diff
@@ -1019,12 +1019,8 @@ svg.icon--stepper-confirmation {
     height: 24px;
     width: 24px;
 }
-svg.icon--stepper-current {
-    height: 24px;
-    width: 24px;
-}
-svg.icon--stepper-information {
-    height: 24px;
+svg.icon--stepper-upcoming {
+    height: 25px;
```

Comment:
This isn't about your changes; I just get a kick out of getting an `svg` of 24x25. 😄

Reply:
Yeah, I tried to get it to be 24x24 but it would be cut off. We can always ask design to update. :)